src/strings/en-GB.json

```diff
@@ -82,6 +82,7 @@
   "LabelUrl": "URL:",
   "LabelZipCode": "Post Code:",
   "LabelffmpegPathHelp": "The path to the ffmpeg application file or folder containing ffmpeg.",
+  "LeaveBlankToNotSetAPassword": "Optional - leave blank to set no password",
```

Comment:
LGTM! One small change, the British translation should only be used for strings that require localization from English such as colour or programme, so a translation isn't required here.

There are probably lots of useless strings in this file still because my script only searched for direct matches to the English translation. Any translations in this file that don't use British terms need to be removed, I was just waiting for Weblate to get set up before I went and removed them all.

Reply:
Fixed.